package.json

```diff
@@ -39,6 +39,7 @@
     "socket.io": "~1.0.6"
   },
   "devDependencies": {
+    "extend": "^3.0.0",
```

Comment:
Mmmh no thanks, a new dependency to fix one test, knowing that the reason itself should be fixed itself ideally (i.e. remove the config read in the link plugin...), let's not go towards that path.